express-zod-api/tests/documentation-helpers.spec.ts

```diff
@@ -127,13 +77,11 @@ describe("Documentation helpers", () => {
 
   describe("depictDefault()", () => {
```

Comment:
all these functions is now the one

Reply:
or maybe these tests should now be transformed into ones of particular postprocessors and assert exectly what they do. So we would not test Zod here, but internals only.

Reply:
moved into todo in the code, will do in another PR